### PR TITLE
Set modules public

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @gkoz :)

### CODE REVIEW

**Comment by gkoz** [comment]
This doesn't seem necessary.

**Comment by GuillaumeGomez** [comment]
Indeed, it's a trait that is needed. I'll export them as well instead.

**Comment by GuillaumeGomez** [comment]
Updated.

File: src/loader.rs [hunk 1]
```diff
@@ -30,8 +30,7 @@ impl PixbufLoader {
             let ptr = ffi::gdk_pixbuf_loader_new_with_type(image_type.to_glib_none().0, &mut error);
             if error.is_null() {
                 Ok(from_glib_full(ptr))
-            }
-            else {
+            } else {
```
Comment (gkoz) [inline src/loader.rs]: What happened here? :)

Comment (GuillaumeGomez) [inline src/loader.rs]: Sorry. >< I can't stand if else statements in C way.

Comment (gkoz) [inline src/loader.rs]: I suppose we could try adopting rustfmt to handle that, but in this commit it's just unwelcome noise.

Comment (GuillaumeGomez) [inline src/loader.rs]: You're hard on business.

**Comment by GuillaumeGomez** [comment]
Done.

**Comment by gkoz** [comment]
Thanks!

**Pull request merged**